pkg/tfbridge/schema_test.go

```diff
@@ -325,6 +326,11 @@ func TestTerraformOutputs(t *testing.T) {
 					},
 				},
 			},
+			"secret_value": {
```

Comment:
Can you add a test for a nested secret property?

Reply:
Sure! Will do :)

Reply:
done!